sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java

```diff
@@ -259,4 +261,41 @@ public void reset() {
             dismissAllowingStateLoss();
         }
     }
+
+    private View prepareRootView(@NonNull LayoutInflater inflater,
```

Comment:
Since this is not reused, why was it extracted into a separate function? Makes code harder to read since you have to jump between methods.

Reply:
To reduce complexity of `onViewCreated` method

Reply:
IMO it reduces complexity of `onViewCreated` but makes code harder to read. I would revert and ignore IDE's recommendation.

Reply:
To be on the same page what exactly "makes overall logic harder to read."?
Methods like `prepareRootView` or `prepareWebView` are really straight forward